### PR TITLE
tasks/cephfs: add TestStrays.test_replicated_delete_speed

### PR DESCRIPTION
**Pull request by jcsp**
Reproducer for http://tracker.ceph.com/issues/16914

Signed-off-by: John Spray <john.spray@redhat.com>

### CODE REVIEW

File: qa/tasks/cephfs/test_strays.py [hunk 1]
```diff
@@ -964,3 +966,33 @@ def unlink_and_count_ops(path, expected_deletions):
...
+
+        self.set_conf("mds.{0}".format(rank_1_id), 'mds_max_purge_files', "0")
+        self.mds_cluster.mds_fail_restart(rank_1_id)
+        self.fs.wait_for_daemons()
+
+        file_count = 10
+
+        self.mount_a.create_n_files("delete_me/file", file_count)
+
+        self._force_migrate(rank_0_id, rank_1_id, "/delete_me",
```
Comment (batrick) [inline qa/tasks/cephfs/test_strays.py]: Where is this method? I couldn't find it anywhere with `git grep`.

Comment (jcsp) [inline qa/tasks/cephfs/test_strays.py]: further up in the file, line 482

Comment (batrick) [inline qa/tasks/cephfs/test_strays.py]: Weird, I don't know why I couldn't find it before. Thanks.

**Comment by batrick** [comment]
`Reviewed-by: Patrick Donnelly <pdonnell@redhat.com>`

**Comment by jcsp** [comment]
This apparently had a merge conflict somewhere along the line and is failing with wrong number of arguments to force_migrate, will fix...

**Comment by jcsp** [comment]
Oh -- it has a conflict with https://github.com/ceph/ceph/pull/15548

**Comment by jcsp** [comment]
Rebased

File: qa/tasks/cephfs/test_strays.py [hunk 2]
```diff
@@ -1011,3 +1013,33 @@ def unlink_and_count_ops(path, expected_deletions):
...
+
+        self.set_conf("mds.{0}".format(rank_1_id), 'mds_max_purge_files', "0")
+        self.mds_cluster.mds_fail_restart(rank_1_id)
+        self.fs.wait_for_daemons()
+
+        file_count = 10
+
+        self.mount_a.create_n_files("delete_me/file", file_count)
+
+        self._force_migrate(rank_1_id, "/delete_me",
```
Comment (batrick) [inline qa/tasks/cephfs/test_strays.py]: should be "delete_me"

Comment (batrick) [inline qa/tasks/cephfs/test_strays.py]: see: http://pulpito.front.sepia.ceph.com/pdonnell-2017-06-27_19:50:40-fs-wip-pdonnell-20170627---basic-smithi/1333833/

Comment (jcsp) [inline qa/tasks/cephfs/test_strays.py]: Oops, updated

**Pull request merged**